src/components/sub-components/company-ui.tsx

```diff
@@ -0,0 +1,92 @@
+import {
```

Comment:
This file should be colocated with the vertical-nav. I suggest we should make a folder inside layout named `vertical-nav` which would contain `vertical-nav.tsx` and `company-ui.tsx`.

Also, a better name for this file would be `workspace-dropdown.tsx`